client/src/assets/scss/main.scss

```diff
@@ -13,19 +13,24 @@
   tab-size: 4;
 }
 
+body {
+  color: #1a1416;
+}
 
-.indexPageList {
-  display:flex;
-  justify-content: center;
-  color:green;
-  
-
-  
-  }
-  
-
-  #restaurantName{
-   border :5px outset green;
+.heading {
+  text-align: center;
+}
 
+.tile {
+  background-color: #453e3f;
+  border-radius: 10px;
+  text-align: center;
+  color: #e6e6e6;
+  box-shadow: 5px 5px 5px #453e3f;
+}
 
-  }
\ No newline at end of file
+.review-tile {
+  border-radius: 10px;
+  text-align: center;
+  box-shadow: 5px 5px 5px #6f645780;
+}
```

Comment:
Especially as apps get larger we can consider other ways to organize our CSS styles, such as separating concerns into smaller SCSS files. There's multiple patterns used in the industry how to organize CSS files - separating by component, by webpage, by type of style (i.e. buttons), etc.

Not something that needs to be refactored here, but something to keep in mind as we get closer to BTs!